### ISSUE TITLE
Forza Motorsport (USA) (En,Ja,Fr,De,Es,It,Pt,Zh,Ko) (v1.70) - Black Screen

### ISSUE THREAD

**Issue by LoveMHz** [MakeMHz/xbox-hd-plus#22]
Forza Motorsport (USA) launches to a black screen. I've confirmed this on a 1.6 system and it's been reported on a 1.1 and a 1.2 system too.

Firmware Version: 2.0.5

**Comment by LoveMHz** [comment]
Well, that was a fun rabbit hole to go down! After messing around I realized very quickly that I was still getting the black screen even with a stock BIOS.

Then I came across this forum post detailing the same issue with a workaround. And it works!
```
Load up MSdash and switch language to any other. Then switch off and on again and go back to ms dash and select original language. Then restart with Forza.
Worked for me first time with HDD and DVD backups
Good Luck.
```
https://forums.afterdawn.com/threads/359980/#post-2285775

This could possibly be an issue related to how the HD+ patch handles the EEPROM config or a game issue. I'm going to leave this issue open until v2.1 is released (which removes the EEPROM workaround for widescreen detection).
